.travis/script.sh

```diff
@@ -31,4 +31,33 @@ else
 		tests/tests
 
 	fi
+
+	# Package and upload non-tagged builds
+	if [ ! -z "$TRAVIS_TAG" ]; then
+		# Skip, handled by travis deploy instead
+		exit 0
+	elif [[ $TARGET_OS == win* ]]; then
+		make -j4 package
+		PACKAGE="$(ls lmms-*win*.exe)"
+	elif [[ $TRAVIS_OS_NAME == osx ]]; then
+		make -j4 install
+		make dmg
+		PACKAGE="$(ls lmms-*.dmg)"
+	else
+		make -j4 install
+		make appimage
+		PACKAGE="$(ls lmms-*.AppImage)"
+	fi
+
+	echo "Uploading $PACKAGE to file.io..."
+	response=$(curl -F "file=@$PACKAGE" "https://file.io")
+	# We need stdout, disable SC2181
+	# shellcheck disable=SC2181
```

Comment:
@zapashcanon FYI

Reply:
I'm wondering if there's another way to do it. Can't figure it out now, but will think about it.

Reply:
I was thinking `if [ foo=$(/usr/bin/true) ];` perhaps.  Haven't tested it and it's a tad bit uglier.

Reply:
Haha, I tried it already, doesn't work. :p (Plus, there's some new warnings from SC because it thinks we want to compare response etc.)

Reply:
OK, actually, it seems to work, but, here's what I got from shellcheck:


```sh
 β ~ cat tmp 
#!/bin/bash

if [ response=$(curl -F "file=@$PACKAGE" "https://file.io") ]; then
	echo cc
fi


 λ ~ ./tmp 
Warning: setting file   failed!
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   145    0     0    0   145      0    145 --:--:-- --:--:-- --:--:--   508
curl: (26) read function returned funny value
cc

 λ ~ shellcheck tmp 

In tmp line 3:
if [ response=$(curl -F "file=@$PACKAGE" "https://file.io") ]; then
     ^-- SC2077: You need spaces around the comparison operator.
              ^-- SC2046: Quote this to prevent word splitting.
```

Reply:
Thanks.  Yes, that's worse.  We'll keep it as-is for now, I guess. https://stackoverflow.com/a/6934931/3196753